NorthstarDLL/plugins/plugins.cpp

```diff
@@ -54,116 +55,69 @@ std::optional<Plugin> PluginManager::LoadPlugin(fs::path path, PluginInitFuncs*
 	Plugin plugin {};
 
 	std::string pathstring = path.string();
-	std::wstring wpath = path.wstring();
+	std::wstring wpath = path.wstring(); // is there any reason to load the libs with wide strings?
```

Comment:
There doesn't seem to be one.
You can replace the call to `LoadLibraryExW` with `LoadLibraryExA` and skip the conversion from `std::string` to `std::wstring`